messaging-experiments.yaml

```diff
@@ -943,3 +943,112 @@
                   action:
                     navigate: true
                     theme: default
+- id: "PRIVACY-RETENTION-STUDY-V2-BUG-1656311"
+  enabled: true
+  filter_expression: (env.channel == 'release' || env.channel == 'default') && env.version >= '79.'
+  # 50% of release
+  targeting: localeLanguageCode == 'en' && [userId, "messaging-experiments-cfr"]|bucketSample(311, 5311, 10000) && 'app.shield.optoutstudies.enabled'|preferenceValue && ((currentDate|date - profileAgeCreated) / 86400000) <= 14
+  arguments:
+    slug: bug-1656311-message-high-velocity-message-testing-privacy-retenti-release-79-80
+    userFacingName: Message Router Content Experiment High Velocity Message Testing Privacy Retention Study V2
+    experimentDocumentUrl: https://experimenter.services.mozilla.com/experiments/high-velocity-message-testing-privacy-retention-study-v2/
+    userFacingDescription: This measures retention changes across some privacy pages including the Protections Dashboard, and Mozilla's Privacy Promise.
+    isEnrollmentPaused: false
+    branches:
+      - slug: control
```

Comment:
Shouldn't slug here match with experimeneter "cfr-to-privacy-promise" and "cfr-to-protections-dashboard" for treatment branch so that experimeneter live dashboard pick right branch, not sure how much it matter

https://experimenter.services.mozilla.com/api/v1/experiments/high-velocity-message-testing-privacy-retention-study-v2/recipe/

Reply:
I don't that matters (as we didn't do this for the v1 experiment), but yeah, let's stick with what experimenter says.